terasoluna-tourreservation-web/src/main/resources/META-INF/spring/applicationContext.xml

```diff
@@ -8,7 +8,24 @@
 
   <import resource="classpath:/META-INF/spring/terasoluna-tourreservation-domain.xml" />
 
-  <bean id="passwordEncoder" class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
+  <bean id="passwordEncoder" class="org.springframework.security.crypto.password.DelegatingPasswordEncoder">
+    <constructor-arg name="idForEncode" value="pbkdf2" />
+    <constructor-arg name="idToPasswordEncoder">
+      <map>
+        <entry key="pbkdf2">
+          <bean class="org.springframework.security.crypto.password.Pbkdf2PasswordEncoder" />
+        </entry>
+        <entry key="bcrypt">
+          <bean class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
+        </entry>
+        <!-- When using SCryptPasswordEncoder, you need to add bcprov-jdk15on.jar to the dependency.
```

Comment:
- [ ]  As comment is unnecessary, let's delete

Reply:
370be5ca97400eb4d3b942128e6185de3ee578f9

I removed in above commit .